engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/details/views/content-host-details.html

```diff
@@ -108,7 +108,7 @@
         <ul uib-dropdown-menu>
           <li>
             <a ui-sref="content-host.packages.actions" translate>
-              Yum Actions
+              Actions
```

Comment:
Should it be 'content' actions? or 'package actions'? or something to indicate what type of actions?

Reply:
There's context based on the parent menu, which is why I just called it Actions, but I could rename it to something else if you think it'd be clearer:

![image](https://user-images.githubusercontent.com/429763/37536040-db8d355c-291f-11e8-9946-fa3205f5115d.png)